package/src/config.ts

```diff
@@ -0,0 +1,5 @@
+import type { ChecklyConfig } from './services/checkly-config-loader'
+
+export function defineConfig(config: ChecklyConfig): ChecklyConfig {
+  return config
+}
```

Comment:
Could you add this to the checkly-config-loader?

Reply:
You mean relocating the helper?